public void testGetThrowableList_Throwable_withoutCause() {
    List<?> throwables = ExceptionUtils.getThrowableList(withoutCause);
    assertEquals(1, throwables.size());
    assertSame(withoutCause, throwables.get(0));
}